### ISSUE TITLE
could not find recipe system for cookbook ruby_rbenv

### ISSUE THREAD

**Issue by ccfiel** [TalkingQuickly/rails-server-template#61]
I followed the steps in quick start in this step ```knife zero converge "name:smsblastr" --ssh-user smsblastr``` I got this error. The error is pasted below. Any idea how to resolve this issue?

```
52.187.119.156   ================================================================================
52.187.119.156   Recipe Compile Error
52.187.119.156   ================================================================================
52.187.119.156   
52.187.119.156   Chef::Exceptions::RecipeNotFound
52.187.119.156   --------------------------------
52.187.119.156   could not find recipe system for cookbook ruby_rbenv
52.187.119.156   
52.187.119.156   System Info:
52.187.119.156   ------------
52.187.119.156   chef_version=13.5.3
52.187.119.156   platform=ubuntu
52.187.119.156   platform_version=16.04
52.187.119.156   ruby=ruby 2.4.2p198 (2017-09-14 revision 59899) [x86_64-linux]
52.187.119.156   program_name=chef-client worker: ppid=108699;start=09:01:46;
52.187.119.156   executable=/opt/chef/bin/chef-client

```

**Comment by flyingboy007** [comment]
getting same error.. Any fixes?

**Comment by tvannurden** [comment]
I skipped the quickstart and did the full version instead. I jumped around a lot but it wasn't too bad

**Comment by flyingboy007** [comment]
@tvannurden will try same. Thank you.

**Comment by TalkingQuickly** [comment]
Hey @flyingboy007 this is now resolved in the repo in version 5.0 along with upgrades to support Ubuntu 20.04, there will be a corresponding new edition to the book in the next few weeks with updated guides.
